src/core/server/saved_objects/service/lib/repository.ts

```diff
@@ -1706,7 +1706,13 @@ export class SavedObjectsRepository {
 
     const normalizedCounterFields = counterFields.map((counterField) => {
       const fieldName = typeof counterField === 'string' ? counterField : counterField.fieldName;
-      const incrementBy = typeof counterField === 'string' ? 1 : counterField.incrementBy || 1;
+      const incrementBy =
+        typeof counterField === 'string'
```

Comment:
nit: Can we please move the nested ternary into a more readable method? This part of the code was already buggy and the more readable it is, the less likely we'll introduce bugs in the future.

Reply:
Updated